pkgs/applications/misc/octoprint/plugins.nix

```diff
@@ -399,6 +399,27 @@ in {
     };
   };
 
+  octolapse = buildPlugin rec {
+    pname = "Octolapse";
+    version = "0.4.1";
+
+    src = fetchFromGitHub {
+      owner = "FormerLurker";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "13q20g7brabplc198jh67lk65rn140r8217iak9b2jy3in8fggv4";
+    };
+
+    propagatedBuildInputs = with super; [ awesome-slugify setuptools pillow sarge six psutil file_read_backwards ];
```

Comment:
```suggestion
    propagatedBuildInputs = with super; [ awesome-slugify setuptools pillow sarge six psutil file-read-backwards ];
```

Reply:
A -full package with all optional plugins isn't a pattern I've seen often in nixpkgs. For example:

https://github.com/NixOS/nixpkgs/tree/master/pkgs/servers/monitoring/grafana
https://github.com/NixOS/nixpkgs/tree/master/pkgs/servers/roundcube
https://github.com/NixOS/nixpkgs/tree/master/pkgs/shells/fish
https://github.com/NixOS/nixpkgs/tree/master/pkgs/applications/networking/cluster/helm

Gimp has plugins, and has a wrapper package `gimp-with-plugins`, but even this defaults to `plugins ? null`
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/graphics/gimp/wrapper.nix

Gimp differs from the above packages in having its plugins exposed as `gimpPlugins.<name>`. Yosys does something similar. This might be less confusing than having a -full package with all plugins, especially given some of these plugins are very situational and not needed by most people. The [ender3v2tempfix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/octoprint/plugins.nix#L150) plugin is essential if and only if you have an ender 3 v2 printer.

Reply:
The name needs to be updated or we run into an eval error when the addon is used.